jazz_deployments/index.js

```diff
@@ -544,4 +548,28 @@ function buildNowRequest(serviceDetails, config, refDeployment) {
 			reject("unable to find deployment details");
 		}
 	});
-}
\ No newline at end of file
+}
+
+const factory = {
+	handler,
+	genericInputValidation,
+	processDeploymentCreation,
+	processDeploymentRebuild,
+	processDeploymentsList,
+	processDeploymentsUpdate,
+	processDeploymentsDeletion,
+	validateDeploymentDetails,
+	addNewDeploymentDetails,
+	validateQueryParams,
+	getDeploymentDetailsByQueryParam,
+	getDeploymentDetailsById,
+	validateUpdateInput,
+	updateDeploymentDetails,
+	deleteServiceByID,
+	reBuildDeployment,
+	getToken,
+	getServiceDetails,
+	buildNowRequest
+}
+
+module.exports = factory;
```

Comment:
Looking at the rest of the code, looks like we can do away with this confusing variable `factory`. Can you remove it and just have that object be directly assigned here?

Reply:
You cannot return the functions directly in module.exports as that cannot be stubbed using sinon(in same module). The way to work around this is to pass an object that returns the functions. If naming is the issue we can come up with another name.

Reply:
Variable name is replaced as mentioned. Second part will be addressed as part of another PR.